proto/zitadel/admin.proto

```diff
@@ -4412,6 +4412,14 @@ message AddSMSProviderTwilioRequest {
             max_length: 200;
         }
     ];
+    string verify_service_sid = 4 [
+        (validate.rules).string = {min_len: 1, max_len: 200},
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "\"AB123b9e61d238abae7d3be7b65ecbc987\"";
+            min_length: 1;
+            max_length: 200;
+        }
+    ];
```

Comment:
since the idea is to have the verification optional, i'd state that in a description and we should either not require a min length or make it an `optional string verify_service_sid`